src/csync/vio/csync_vio_local_unix.cpp

```diff
@@ -129,22 +130,33 @@ std::unique_ptr<csync_file_stat_t> csync_vio_local_readdir(csync_vio_handle_t *d
   }
 #endif
 
-  return file_stat;
+    if (file_stat->path.isNull())
+        return file_stat;
+
+    if(_csync_vio_local_stat_mb(fullPath.constData(), file_stat.get()) < 0) {
+        file_stat->type = CSYNC_FTW_TYPE_SKIP;
```

Comment:
There was a log in `csync_vio_stat` in that case that could be useful, maybe it could be added  in `_csync_vio_local_stat_mb`.
Also the comment `// Will get excluded by _csync_detect_update.` from `csync_ftw ` could be preserved/rewritten here.